Cython/Utility/ObjectHandling.c

```diff
@@ -196,12 +196,14 @@ static CYTHON_INLINE PyObject *__Pyx_PyIter_Next2(PyObject* iterator, PyObject*
     // We always do a quick slot check because calling PyIter_Check() is so wasteful.
     iternextfunc iternext = Py_TYPE(iterator)->tp_iternext;
     if (likely(iternext)) {
-#if CYTHON_USE_TYPE_SLOTS
+#if CYTHON_COMPILING_IN_PYPY || CYTHON_USE_TYPE_SLOTS
```

Comment:
Sorry for sounding a bit pedantic here, but I'd reverse these two to have the more generic option first.
```suggestion
#if CYTHON_USE_TYPE_SLOTS || CYTHON_COMPILING_IN_PYPY
```

Reply:
done